api/src/Entity/ContentNode.php

```diff
@@ -67,6 +68,10 @@ abstract class ContentNode extends BaseEntity implements BelongsToContentNodeTre
     #[Assert\NotNull(groups: ['create'])] // Root nodes have parent:null, but manually creating root nodes is not allowed
     #[AssertNoRootChange(groups: ['update'])]
     #[AssertNoLoop(groups: ['update'])]
+    #[AssertInstanceOf(
+        type: SupportsContentNodeChildren::class,
+        message: 'This parent does not support children, only content_nodes of type column_layout support children.'
```

Comment:
i tried to write the type like it's exposed in the api url, but no hard feelings if we use the php class name.
But i would then use ContentNode instead of content_node.

Reply:
Nevermind, also ok for me